source/SIL.AppBuilder.Portal.Frontend/package.json

```diff
@@ -23,13 +23,16 @@
     "typecheck": "yarn tsc --noEmit"
   },
   "dependencies": {
+    "@material-ui/core": "^3.1.0",
+    "@material-ui/icons": "^3.0.1",
     "@orbit/coordinator": "^0.15.14",
     "@orbit/core": "^0.15.14",
     "@orbit/indexeddb": "^0.15.14",
     "@orbit/indexeddb-bucket": "^0.15.14",
     "@orbit/jsonapi": "^0.15.15",
     "@orbit/local-storage-bucket": "^0.15.14",
     "@orbit/store": "^0.15.14",
+    "add": "^2.0.6",
```

Comment:
what's this for?

Reply:
not sure... don't remember adding it. I will remove it

Reply:
Ok... I know what happened. I did yarn add add @material-ui/icons, funny that is an "add" package in npm wonder how many download it had just because typos like mine